fideslog/sdk/python/client.py

```diff
@@ -66,33 +86,45 @@ def __init__(
         self.developer_mode = developer_mode
         self.extra_data = extra_data or {}
 
-    def send(self, event: AnalyticsEvent) -> None:
+    def register(self, registration: Registration) -> None:
```

Comment:
It'd be good to also support registration [removal and editing](https://github.com/ethyca/fideslog/pull/87/files#diff-61dce5a1fdfd0c8e3335ab76e6e1e8e14344e9354ab490b5f0f747b2ec17f6c2R101-R151) here in the client as a follow-up, that would enable Fides keep their registration info up to date in Fideslog.

Reply:
Created #91 to track this work.